analyses/molecular-subtyping-HGG/results/HGG_cleaned_all_table.tsv

```diff
@@ -795,7 +795,7 @@ PT_BC0B16HC	BS_PYYNXW86	BS_1D93Z575	BS_ENH6KT2R	7316-7075	7316-7075_Solid Tissue
 PT_0324HWD5	NA	BS_QSS96VAM	NA	7316-7091	7316-7091_Not Available		8	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	Not Available	NA	NA	NA	NA	NA	None	None	None	None	None	None	0	NA	HGG, To be classified
 PT_0324HWD5	BS_Q3H2JNNM	BS_YR1XY6SF	NA	7316-7091	7316-7091_Solid Tissue	Hemispheric	8	No	No	No	No	No	No	No	No	No	No	BRAF	No R132 or R172	None	No V600E	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	Solid Tissue	1.4198035337457413	0.0099959300739331	0.2599601124035604	-0.39704099869214593	-0.5810301482426811	None	None	None	None	None	None	0	NA	HGG, H3 wildtype
 PT_7S5BVPE2	NA	NA	BS_548VV6P6	7316-712	7316-712_Solid Tissue	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	Solid Tissue	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	DMG, H3 K28	DMG, H3 K28
-PT_FCWMT41S	BS_XK482X70	NA	BS_4Q7PQHS8	7316-7133	7316-7133_Solid Tissue	Midline	5	No	No	No	No	No	No	No	No	No	No	None	No R132 or R172	None	No V600E	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	Solid Tissue	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	DMG, H3 K28	DMG, H3 K28
+PT_FCWMT41S	BS_XK482X70	NA	BS_4Q7PQHS8	7316-7133	7316-7133_Solid Tissue	Midline	5	No	No	No	No	No	No	No	No	No	No	None	No R132 or R172	None	No V600E	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	neutral	Solid Tissue	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	DMG, EGFR	HGG, H3 wildtype
```

Comment:
you might need another rule that if there is no mutation, take DMG, EGFR and make it DMG, K28 as the molecular subtype. I see a few being converted back to wildtype, but we should just include this when we do that final search using methyl only.

Reply:
Thank you for pointing out. This issue is addressed.